core/app/views/brightcontent/base/_page_sizes.html.erb

```diff
@@ -0,0 +1,9 @@
+<% if sizes.try(:any?) %>
```

Comment:
`per_page_sizes` defaults to an array so the nil check isn't necessary. Also, if there is only one element these options don't need to be shown either.